governance/polls/SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024.md

```diff
@@ -0,0 +1,55 @@
+---
+title: SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024
```

Comment:
:yellow_circle: **minor**: capitalize words in title. Remove value from title to maintain consistency with https://vote.makerdao.com/polling/QmYYoAMe#poll-detail

```suggestion
title: SparkLend Ethereum - Increase ETH Borrow Cap gap Parameter - March 25, 2024
```

Reply:
Also add capitals to file name if changing